ui/css/module/_footnotes.scss

```diff
@@ -5,14 +5,38 @@
   padding: 2px 4px;
   font-size: 0.6em;
   margin-left: .3em;
+  position: relative;
 
   &.active {
-    border-color: $color-primary-darker;
-    background: $color-primary-darker;
-    color: $color-white;
+    @extend .active-footnote-style;
+
+    &::before {
+      @extend .rotated-triangle;
+      z-index: 10;
+      content: '';
+      position: absolute;
+    }
   }
 }
 
+.active-footnote-style {
+  border-color: $color-primary-darker;
+  background: $color-primary-darker;
+  color: $color-white;
+}
+
+.rotated-triangle {
+  top: 39px;
```

Comment:
Is there a way to compute this from other values (maybe based on the margin value of the inline footnote?) so it is less of a magic number?